webapp/platform/client/src/websocket.ts

```diff
@@ -99,7 +99,7 @@ export default class WebSocketClient {
         // Add connection id, and last_sequence_number to the query param.
         // We cannot use a cookie because it will bleed across tabs.
         // We cannot also send it as part of the auth_challenge, because the session cookie is already sent with the request.
-        this.conn = new WebSocket(`${connectionUrl}?connection_id=${this.connectionId}&sequence_number=${this.serverSequence}`);
+        this.conn = new WebSocket(`${connectionUrl}?connection_id=${this.connectionId}&sequence_number=${this.serverSequence}&posted_ack=true`);
```

Comment:
Instead of being hardcoded, I think this should be specified in either the WS constructor or as a parameter to initialize since we can't guarantee that everyone who uses `@mattermost/client` will ack posted messages

Reply:
That is a good point. I was only thinking web-based stuff would use this, but since we publish this separately I'll make it a parameter.